src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write your code here
+const logo = document.getElementsByClassName('logo')[0];
+
+const promise1 = new Promise((resolve, reject) => {
+  const newDiv = document.createElement('div');
+  const newContent = document.createTextNode('Promise was resolved!');
+
+  newDiv.className = 'message';
+  newDiv.appendChild(newContent);
+
+  const body = document.body;
+
+  logo.addEventListener('click', () => {
+    resolve(body.insertAdjacentElement('afterbegin', newDiv));
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  const newDiv = document.createElement('div');
+  const newContent = document.createTextNode('Promise was rejected!');
+
+  newDiv.className = 'message error-message';
+
+  newDiv.appendChild(newContent);
+
+  const body = document.body;
+
+  setTimeout(() => {
+    logo.addEventListener('click', () => {
+      reject(body.insertAdjacentElement('afterbegin', newDiv));
+    });
+  }, 3000);
+});
+
+promise1.then((resolve) => resolve);
+promise2.then((reject) => reject);
+
+Promise.allSettled([
+  promise1.then((resolve) => resolve),
+  promise2.then((reject) => reject),
+]);
```

Comment:
The `Promise.allSettled` method is used to handle multiple promises, but the `then` methods inside it are not necessary. `Promise.allSettled` will return an array of results, each being an object with a `status` and `value` or `reason`. Consider handling the results of `Promise.allSettled` to perform actions based on whether each promise was fulfilled or rejected.